src/lib/arch/aarch64/Architecture.cc

```diff
@@ -14,13 +14,14 @@ Architecture::Architecture(kernel::Linux& kernel, ryml::ConstNodeRef config)
       SVL_(config["Core"]["Streaming-Vector-Length"].as<uint64_t>()),
       vctModulo_((config["Core"]["Clock-Frequency-GHz"].as<float>() * 1e9) /
                  (config["Core"]["Timer-Frequency-MHz"].as<uint32_t>() * 1e6)) {
-  if (cs_open(CS_ARCH_ARM64, CS_MODE_ARM, &capstoneHandle_) != CS_ERR_OK) {
+  if (cs_open(CS_ARCH_AARCH64, CS_MODE_ARM, &capstoneHandle_) != CS_ERR_OK) {
     std::cerr << "[SimEng:Architecture] Could not create capstone handle"
               << std::endl;
     exit(1);
   }
 
   cs_option(capstoneHandle_, CS_OPT_DETAIL, CS_OPT_ON);
+  cs_option(capstoneHandle_, CS_OPT_DETAIL, CS_OPT_DETAIL_REAL);
```

Comment:
This option reverts the aliasing so that operand details etc are correct and eliminates the need for `revertAlias()` in `instructionMetadata.cc`. 

Currently, the mnemonic and operand string in the metadata will still be that of the alias, but in the exceptionHandler printout there is a message stating this and the Capstone developers are currently working to fix this.

Reply:
Comment added to provide some explanation